ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -206,7 +207,16 @@ def get_column_tag_labels(
         return None
 
     def set_inspector(self, database_name: str):
-        self.client = Client(project=database_name)
+        quota_project_id = self.service_connection.connectionOptions.get("quotaProjectId", None)
+        location = self.service_connection.connectionOptions.get("location", None)
+        impersonate_service_account = self.service_connection.connectionOptions.get("impersonateServiceAccount", None)
+        lifetime = self.service_connection.connectionOptions.get("lifetime", None)
+        self.client = get_bigquery_client(
+            quota_project_id=quota_project_id,
+            project_id=database_name,
+            impersonate_service_account=impersonate_service_account,
+            location=location,
+            lifetime=lifetime)
```

Comment:
I would like to discuss if we should implement the input entity or not, instead of the connection options.

Reply:
It indeed might be helpful to have these parameters defined in the JSON Schema so that we can give proper descriptions and guidelines to users

Reply:
Alright. I will give it a try to change the JSON schema.